tests/providers/google/cloud/hooks/test_dataprep.py

```diff
@@ -0,0 +1,88 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from unittest import TestCase, mock
+
+from mock import patch
+from requests import HTTPError
+from tenacity import RetryError
+
+from airflow.providers.google.cloud.hooks import dataprep
+
+JOB_ID = 1234567
+URL = "https://api.clouddataprep.com/v4/jobGroups/1234567/jobs"
+
+
+class TestGoogleDataprepHook(TestCase):
+    def setUp(self):
+        self.hook = dataprep.GoogleDataprepHook(dataprep_conn_id="dataprep_conn_id")
+
+    @patch("airflow.providers.google.cloud.hooks.dataprep.requests.get")
+    @patch("airflow.providers.google.cloud.hooks.dataprep.GoogleDataprepHook._token")
+    def test_mock_should_be_called_once_with_params(self, mock_hook, mock_get_request):
```

Comment:
```suggestion
    def test_mock_should_be_called_once_with_params(self, mock_token, mock_get_request):
```
This may require better mocking with PropertyMock

Reply:
Fixed and I will check PropertyMock.
Thank you.